notebooks/test/executed_notebooks/Storage Client Library_output.ipynb

```diff
@@ -0,0 +1,391 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext google.cloud.bigquery"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Google Cloud Storage\n",
+    "\n",
+    "This page shows how to get started with the Google Cloud Storage Python client library."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import the library"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from google.cloud import storage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Projects\n",
+    "\n",
+    "All data in Cloud Storage belongs inside a project. A project consists of a\n",
+    "set of users, a set of APIs, and billing, authentication, and monitoring\n",
+    "settings for those APIs. You can have one project or multiple projects."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Clients\n",
+    "Start by initializing a client, which is used to interact with the Google Cloud Storage API. The project used by the client will default to the project associated with your credentials. Alternatively, you can explicitly specify a project when constructing the client."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = storage.Client(project=\"ajhamilton-scratch\")"
```

Comment:
We probably don't want to check in filled-in project IDs even for executed output, right?

Reply:
Updated the test run to store executed notebooks in a gitignored folder and wipe the project IDs from the rendered executed files.